homepage/blocks/homepage-link-bar/homepage-link-bar.css

```diff
@@ -0,0 +1,121 @@
+.homepage-link-bar {
+  padding: var(--spacing-s) var(--spacing-s) 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  text-size-adjust: none;
+}
+
+.homepage-link-bar > div {
+  flex-basis: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.homepage-link-bar.custom-bg {
+  padding-bottom: var(--spacing-s);
+}
+
+.homepage-link-bar h2 {
+  margin: var(--spacing-xxs);
+  line-height: var(--spacing-s);
+  font-size: 20px;
+  text-align: center;
+}
+
+.homepage-link-bar a.con-button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: var(--spacing-xxs);
+  width: auto;
+  padding: 0 14px;
+  min-height: 26px;
+  white-space: nowrap;
+}
+
+.homepage-link-bar a.con-button.outline:hover svg path {
+  fill: #fff;
+}
+
+.homepage-link-bar a.con-button svg {
+  width: auto;
+  height: auto;
+  margin-top: 4px;
+}
+
+.homepage-link-bar.justified > div > div {
+  flex-basis: 100%;
+}
+
+@media screen and (min-width: 600px) and (max-width: 1024px) {
+  .homepage-link-bar > div {
+    flex-basis: 0;
+  }
+
+  .homepage-link-bar.button-count-5 div.button-index-1,
+  .homepage-link-bar.button-count-5 div.button-index-2 {
+    flex-basis: 50%;
+    justify-content: flex-end;
+  }
+
+  .homepage-link-bar.button-count-5 div.button-index-2 {
+    justify-content: flex-start;
+  }
+
+  .homepage-link-bar.button-count-6 div:not(.header) {
+    flex-basis: 33%;
+    justify-content: flex-end;
+  }
+
+  .homepage-link-bar.button-count-6 div.button-index-2,
+  .homepage-link-bar.button-count-6 div.button-index-5 {
+    flex-basis: 0;
+  }
+
+  .homepage-link-bar.button-count-6 div.button-index-3,
+  .homepage-link-bar.button-count-6 div.button-index-6 {
+    justify-content: flex-start;
+  }
+
+  .homepage-link-bar > div.header {
+    flex-basis: 100%;
+  }
+
+  .homepage-link-bar.justified > div > div {
+    flex-basis: auto;
+  }
+}
+
+@media screen and (min-width: 1025px) {
+  .homepage-link-bar {
+      flex-wrap: nowrap;
+  }
+
+  .homepage-link-bar.button-count-6 {
+      flex-wrap: wrap;
+  }
+
+  .homepage-link-bar > div {
+    flex-basis: content;
+  }
+
+  .homepage-link-bar > div.header {
+    white-space: nowrap;
+  }
+
+  .homepage-link-bar.button-count-6 > div.header {
+    flex-basis: 100%;
+  }
+}
+
+@media screen and (min-width: 1300px) {
```

Comment:
I see other sites use 1200px for desktop, do we have standard breakpoints in our sites? Maybe consonant?

Reply:
Well, the 1300px breakpoint was only for the fringe case of 6 buttons in the block.  It needed to wrap a little sooner to still look good.  The real issue was the media query above it for 1025px.  That's when "desktop" was starting.  But you're right so  I moved that breakpoint to 1200.  But I'd like to keep the 1300px additional breakpoint for 6 buttons.

Reply:
Here's a preview page a made a while back to see the behavior of the block with different numbers of buttons: https://homepage-brick--homepage--adobecom.hlx.page/homepage/drafts/vgoodrich/link-bars